install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -500,7 +500,9 @@ data:
 {{- end }}
 
 {{- if .Values.eni.enabled }}
+  {{- if not (and .Values.endpointRoutes .Values.endpointRoutes.enabled) }}
```

Comment:
Unless I'm missing something, I believe the `and` is not necessary here: Helm should correctly parse `if not .Values.endpointRoutes.enabled` correctly even if `.Values.endpointRoutes` is unset

Reply:
Yup, you're right! Validated via local template, the main object check is unnecessary. Fixed!